antarest/matrixstore/matrix_uri_mapper.py

```diff
@@ -166,7 +166,7 @@ def denormalize(self, node: MatrixNode) -> None:
 
         uuid = link_path.read_text()
         matrix = self.get_matrix(uuid)
-        node.dump(matrix)
+        node.write_dataframe(matrix)
```

Comment:
We should probably also use that serialization method for the matrix content repo when CSV is configured.
We can have a common method somewhere in `core.serde`.